target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -256,6 +256,9 @@ ramips_board_detect() {
 	*"M4 (8M)")
 		name="m4-8M"
 		;;
+	*"Maestro E220")
```

Comment:
There is no need to keep here whole model name, `E220` is enough. Moreover, general convention is to use only board name, without manufacturer name (take ZyXEL devices as example).